langfuse-core/src/index.ts

```diff
@@ -674,14 +687,207 @@ abstract class LangfuseCoreStateless {
     }
   }
 
+  protected async processMediaInEvent(type: LangfuseObject, body: EventBody): Promise<void> {
+    if (!body) return;
+
+    const traceId = "traceId" in body ? body.traceId : type.includes("trace") ? body.id : undefined;
+
+    if (!traceId) {
+      throw new Error("traceId is required for media upload");
```

Comment:
Whya re you throwing here? I would just return as this is not an error condition, right?